client/src/components/_common/Button/Button.module.css

```diff
@@ -35,6 +35,10 @@
   composes: c-button--as-link from '../../../styles/components/c-button--new.css';
 }
 
+.root.as-link {
+  font-size: 1rem;
+}
+
```

Comment:
⚠️ This change affects at least one other view (move/copy modal), because this style applies to **all** `.as-link` buttons.

https://github.com/user-attachments/assets/4e089e62-25fa-4b1a-8385-668da95cbbbf